contracts/GenArt721CoreV3.sol

```diff
@@ -230,6 +237,14 @@ contract GenArt721CoreV3 is
         _updateRandomizerAddress(_randomizerContract);
         // set AdminACL management contract as owner
         _transferOwnership(_adminACLContract);
+        // initialize contract base URI
+        _updateContractBaseURI(
+            string.concat(
+                "https://token.artblocks.io/",
+                address(this).toHexString(),
+                "/"
+            )
+        );
```

Comment:
Is this codifying an assumption that V3 needs to have a base-URI that includes the contract address rather than one that just supports `https://token.artblocks.io/{TOKEN_ID}`?

I don't love that, and think that's not a good path forward (I think that `https://token.artblocks.io/{TOKEN_ID}` fallback _needs_ to work properly for Art Blocks flagship).

If that isn't the case, I would suggest here that the default assignment on construction should be just `https://token.artblocks.io/` (no concatenating the address).

Reply:
The field is admin-updateable, so the value during deployment is not permanent.

Including the address was based on quick convo w/ @Jbern16 yesterday, but definitely can remove the addr here if we don't want to go in that direction for flagship. There is a slight benefit of including the addr in the default token URI during deployment because it would be common between flagship and AB engine contracts, but also totally reasonable to have difference in the contracts here.

Reply:
Makes sense to me–I think if we just make this `https://token.artblocks.io/`, I'm less concerned about the assumptions of this PR because it forces us to handle this backwards compatibility in our infra.

If we want to have address injected, then I think we need to discuss this a bit more...

> There is a slight benefit of including the addr in the default token URI during deployment because it would be common between flagship and AB engine contracts, but also totally reasonable to have difference in the contracts here.

I'd say this should just be one of the differences between V3 and V4 (new partner/engine standard).

Reply:
SGTM - updated to just be `https://token.artblocks.io/` in constructor, no address.

Cc @Jbern16 @lyaunzbe

Reply:
this decision sgtm fwiw 👍